nms/package.json

```diff
@@ -70,5 +70,8 @@
     "packages": [
       "packages/*"
     ]
+  },
+  "resolutions": {
+    "**/@fbcnms/**/@fbcnms/sequelize-models": "^0.1.13"
```

Comment:
The syntax is weird, I'm not sure if it can be simplified :) See https://github.com/yarnpkg/yarn/issues/4874#issuecomment-601452256

Slash is a separator for parent -> child package resolution, and add the same time a separator for the namespaces :)

As I understand the syntax, it says:
For any package that matches `**/@fbcnms/**` use `@fbcnms/sequelize-models: "^0.1.13"`